app/views/layouts/_bugsnag_js.html.haml

```diff
@@ -0,0 +1,5 @@
+- bugsnag_js_key = ENV['BUGSNAG_JS_KEY'] || ENV['BUGSNAG_API_KEY']
+- if bugsnag_js_key.present?
+  %script{src: "//d2wy8f7a9ursnm.cloudfront.net/v6/bugsnag.min.js"}
```

Comment:
I just had a call to discuss exactly this on another project :see_no_evil: We can't use a package manager I guess, right? We'll surely forget to upgrade it manually but I guess that's life :man_shrugging:

Reply:
![Screenshot from 2020-03-31 13-08-19](https://user-images.githubusercontent.com/9029026/78019949-d10ba280-7350-11ea-9d51-f9715738158e.png)

:sob:

Reply:
Close but no cigar 🚬

Reply:
That's even more than I expected @Matt-Yorkley ! let's just create an issue for it then.

Reply:
We can't have _any_ nice things until we get to Rails 4...

![KEEP CALM](https://user-images.githubusercontent.com/9029026/78048906-adf5e880-737a-11ea-87c9-fa90b79e3b75.png)